lib/pc_schema.c

```diff
@@ -594,13 +562,21 @@ pc_schema_from_xml(const char *xml_str, PCSCHEMA **schema)
 		}
 	}
 
-	xmlXPathFreeObject(xpath_obj);
+	if( pc_schema_is_valid(s) )
```

Comment:
Space after `if`. (The tools should enforce our code styling but this is not the case currently.)